workflows/taxprofiler.nf

```diff
@@ -144,7 +144,7 @@ workflow TAXPROFILER {
         }
 
     // Merge ch_input.fastq and ch_input.nanopore into a single channel
-    def ch_input_for_fastqc = ch_input.fastq.mix( ch_input.nanopore )
+    ch_input_for_fastqc = ch_input.fastq.mix( ch_input.nanopore )
```

Comment:
I assume it was working fine with the `def`, why the change?

Reply:
For consistency with the rest of the pipeline - at no other point do we use `def` to specify a channel (only specifically for Groovy related stuff), and to my knowledge it's not needed.

Is there a benefit of it do you know?

Reply:
Not huge, it defines the variable within the current scope. Otherwise it ends up being a global variable. Normally shouldnt be a problem but might have unintended consequences.